packer/http/centos-7.2/ks-openstack.cfg

```diff
@@ -0,0 +1,86 @@
+install
+cdrom
+lang en_US.UTF-8
+keyboard us
+network --bootproto=dhcp
+rootpw centos
+firewall --disabled
+selinux --permissive
+timezone UTC
+unsupported_hardware
+bootloader --location=mbr
+text
+skipx
+zerombr
+clearpart --all --initlabel
+part / --fstype="ext4" --grow --size=100
+auth --enableshadow --passalgo=sha512 --kickstart
+firstboot --disabled
+reboot
+user --name=centos --plaintext --password centos
+url --url=http://centos.osuosl.org/7/os/x86_64/
+repo --name=updates --baseurl=http://centos.osuosl.org/7/updates/x86_64/
```

Comment:
Let's go ahead and statically set this to ``7.2`` so it always builds that version.